test/sequential/test-http-max-http-headers.js

```diff
@@ -122,8 +122,8 @@ const test2 = common.mustCall(() => {
     'Agent: nod2\r\n' +
     'X-CRASH: ';
 
-  // /, Host, localhost, Agent, node, X-CRASH, a...
-  const currentSize = 1 + 4 + 9 + 5 + 4 + 7;
+  // Host, localhost, Agent, node, X-CRASH, a...
+  const currentSize = 4 + 9 + 5 + 4 + 7;
```

Comment:
Can you please explain this change?

Reply:
currentSize is being used by `fillHeaders` to subtract from the max header size and generate a header value with a length that goes over the limit. Since the implementation at node_http_parser_impl.h when using llhttp doesn't count the url size towards the header size after the changes, there's no point considering the "/" size when using `fillHeaders` anymore.